big-int/src/helpers.cpp

```diff
@@ -29,10 +29,49 @@
 #include <ostream>
 #include <bigint.hpp>
 #include <algorithm>
+#include <vector>
+#include <complex>
 
 namespace libbig
 {
     int char_int_converter(const char &x) {
         return ((x >= '0' && x <= '9') ? x - '0' : x + '0') ;
     }
+
+    complexCoeffs fast_fourier_transform(const bool is_IDFT, const complexCoeffs num) {
```

Comment:
Codacy found an issue: [Function parameter 'num' should be passed by const reference.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6394972)